src/main/java/de/srendi/advancedperipherals/common/addons/computercraft/peripheral/InventoryManagerPeripheral.java

```diff
@@ -55,7 +55,7 @@ public final int addItemToPlayer(String inventoryBlock, int count, Optional<Stri
         IItemHandler inventoryFrom = targetEntity != null ? targetEntity
                 .getCapability(CapabilityItemHandler.ITEM_HANDLER_CAPABILITY, direction).resolve().orElse(null) : null;
         PlayerInventory inventoryTo = getOwnerPlayer().inventory;
-        if (inventoryFrom == null) {
+        if (inventoryFrom == null || inventoryTo == null) {
```

Comment:
You can remove `inventoryFrom == null`. We already check this via `ensurePlayerLinked()`.
And could you remove the debug as well?

Reply:
Done